FCCee/compact/CLD/CLD_o3_v01/README.md

```diff
@@ -0,0 +1,38 @@
+CLD_o3_v1 Detector
+======================
+
+This option (o3) of CLD includes the Array of RICH Cells (ARC) subdetector. The ARC is placed after the tracker and before the calorimeters. 
```

Comment:
Did I miss the includes, or is the ARC_o1_v1.xml detector _only_ the ARC, and nothing else of CLD?

Reply:
For now yes, the full configuration is being prepared and will be added in a second PR. The point is that the ARC detector has been optimized for a certain radius and for now we can not easily change it which means that we have to shrink the CLD tracker by ~20 cm, which will probably impact the performance significantly...